src/program/calibration.rs

```diff
@@ -115,6 +115,19 @@ impl CalibrationSet {
                                     }
                                 }
                             }
+                            if let Instruction::Delay(Delay { qubits, .. }) = instruction {
```

Comment:
Can you switch this to a `match` and incorporate the `Gate`, now that it's the second case to handle?

In fact, they can be the same match arm with `|`, since `qubits` is of the same type and the code is the same.

Reply:
Great call! See fb04b8f